backend/src/test/repository/DiscountRepository.test.ts

```diff
@@ -24,7 +24,7 @@ describe('Unit tests for DiscountRepository', function () {
       fail('Should have failed');
     } catch (e) {
       expect(e.message).toBe(
-        'null value in column "promotionId" violates not-null constraint'
+        'null value in column "promotionId" of relation "discount" violates not-null constraint'
```

Comment:
Hmm strange, was this failing before?

Reply:
Yeah it was for some reason!

Reply:
Hmm does this happen on `master` branch as well?